doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -191,6 +191,41 @@ name "/Q30/"
 
 \begin_layout Labeling
 \labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/Q40/"
+
+\end_inset
+
+ Beagle needs a define time to run in dependencey of the 
```

Comment:
As discussed today, this is almost impossible to specifiy. Because runtime could be exponential in some cases and linear in others.